src/Sylius/Behat/Context/Api/Shop/PaymentRequestContext.php

```diff
@@ -81,4 +92,33 @@ public function putPaymentRequest(string $paymentRequestUri, array $payload = []
 
         $this->client->executeCustomRequest($request);
     }
+
+    /**
+     * @Then my payment request with action :action for payment method :paymentMethod should have state :state
+     */
+    public function myPaymentRequestShouldBeCancelled(string $action, PaymentMethodInterface $paymentMethod, string $state): void
+    {
+        if ($this->getRequestForPaymentRequestWithAction($action) === null) {
+            throw new \InvalidArgumentException(sprintf('Payment request with action "%s" not found', $action));
+        }
+
+        $request = $this->getRequestForPaymentRequestWithAction($action);
```

Comment:
```suggestion
        $request = $this->getRequestForPaymentRequestWithAction($action);

        if (null === $request) {
            throw new \InvalidArgumentException(sprintf('Payment request with action "%s" not found', $action));
        }

        
```

Reply:
Addin to that, `Assert::notNull` or `Assert::instanceOf` with a custom message would suffice